q2_boots/plugin_setup.py

```diff
@@ -380,3 +383,102 @@
+        ('kmer_tables', Collection[FeatureTable[Frequency]]),
+        ('alpha_diversities', Collection[SampleData[AlphaDiversity]]),
+        ('distance_matrices', Collection[DistanceMatrix]),
+        ('pcoas', Collection[PCoAResults]),
+        ('scatter_plot', Visualization),
+    ],
+    input_descriptions={'table': _feature_table_description,
+                        'sequences': "Input sequences for kmerization."},
+    parameter_descriptions={
+        'metadata': 'The sample metadata used in generating Emperor plots.',
+        'n': _n_description,
+        'sampling_depth': _sampling_depth_description,
+        'alpha_average_method': 'Method to use for averaging alpha diversity.',
+        'beta_average_method': 'Method to use for averaging beta diversity.',
+        'replacement': _replacement_description,
+        'kmer_size': 'Length of kmers to generate.',
+        'tfidf': 'If True, kmers will be scored using TF-IDF and output '
+                 'frequencies will be weighted by scores. If False, kmers are '
+                 'counted without TF-IDF scores.',
+        'max_df': 'Ignore kmers that have a frequency strictly higher than '
+                  'the given threshold. If float, the parameter represents a '
+                  'proportion of sequences, if an integer it represents an '
+                  'absolute count.',
+        'min_df': 'Ignore kmers that have a frequency strictly lower than '
+                  'the given threshold. If float, the parameter represents a '
+                  'proportion of sequences, if an integer it represents an '
+                  'absolute count.',
+        'max_features': 'If not None, build a vocabulary that only considers '
+                        'the top max_features ordered by frequency (or TF-IDF '
+                        'score).',
+        'norm': 'Normalization procedure applied to TF-IDF scores. Ignored '
+                'if tfidf=False. l2: Sum of squares of vector elements is 1. '
+                'l1: Sum of absolute values of vector elements is 1.',
+        'pc_dimensions': 'Number of principal coordinate dimensions to keep '
+                         'for plotting.',
+        'color_by': 'Categorical measure from the input Metadata that '
+                    'should be used for color-coding the scatterplot.'
+    },
+    output_descriptions={
+        'resampled_tables': _resampled_tables_description,
+        'kmer_tables': 'The `n` kmer tables.',
+        'alpha_diversities': 'Average alpha diversity vector for each metric.',
+        'distance_matrices': ('Average beta diversity distance matrix for '
+                              'each metric.'),
+        'pcoas': 'PCoA matrix for each beta diversity metric.',
+        'scatter_plot': ('Scatter plot including alpha diversity and '
+                         'pcoa results for all selected metrics.')
+    },
+    name='Perform resampled "core metrics" analysis.',
```

Comment:
The name is incorrect. Instead: "Perform resampled kmer-based diversity analysis with kmerizer"?